store/storetest/team_store.go

```diff
@@ -416,7 +416,9 @@ func testTeamStoreGetByIniviteId(t *testing.T, ss store.Store) {
 	}
 
 	o2.InviteId = ""
-	<-ss.Team().Update(&o2)
+	if err := ss.Team().Update(&o2); err != nil {
```

Comment:
We prefer the `require.Nil(t, err)` (I know we are using them inconsistently, but we want to "migrate" to the require approach eventually).